spanner_orm/tests/model_test.py

```diff
@@ -200,6 +200,62 @@ def test_validation(self):
           skip_validation=False,
       )
 
+  def test_model_equates(self):
+    timestamp = datetime.datetime.now(tz=datetime.timezone.utc)
+    test_model1 = models.UnittestModel({
+        'int_': 0,
+        'float_': 0,
+        'string': '',
+        'string_array': ['foo', 'bar'],
+        'timestamp': timestamp,
+    })
+    test_model2 = models.UnittestModel({
+        'int_': 0,
+        'float_': 0.0,
+        'string': '',
+        'string_array': ['foo', 'bar'],
+        'timestamp': timestamp,
+    })
+    self.assertEqual(test_model1, test_model2)
+
+  @parameterized.parameters(
+      (models.UnittestModel({
+          'int_': 0,
+          'float_': 0,
+          'string': '1',
+          'timestamp': datetime.datetime.now(tz=datetime.timezone.utc),
```

Comment:
(nit) `datetime.datetime.now()` can return a different value every time it's called, or not. I'd make a private module-level constant for this, to avoid any issues with flakiness (and to test the thing you're trying to test instead of the timestamps).

Reply:
Fair